FirebaseAuth.podspec

```diff
@@ -60,8 +60,8 @@ supports email and password accounts, as well as several 3rd party authenticatio
       'FIRAuth_VERSION=' + s.version.to_s +
       ' FIRAuth_MINOR_VERSION=' + s.version.to_s.split(".")[0] + "." + s.version.to_s.split(".")[1]
   }
-  s.framework = 'SafariServices'
   s.framework = 'Security'
+  s.ios.framework = 'SafariServices'
```

Comment:
Sorry I don't understand why this restores `SafariServices`.
Will it affect other platforms?

Reply:
In the previous version with :
```
  s.framework = 'SafariServices'
  s.framework = 'Security'
```
The value of s.framework was set to `SafariServices` and then overwritten with `Security` before it was processed.  

If you look at the build FirebaseAuth-iOS Build target in a previous build of Firebase.xcworkspace, you'll see that SafariServices was missing:

<img width="1180" alt="screen shot 2018-10-24 at 4 32 51 pm" src="https://user-images.githubusercontent.com/73870/47467303-a3f5b080-d7aa-11e8-8be0-d98b8bba97c0.png">

Reply:
As I discovered in travis testing, SafariServices doesn't exist on tvOS and we don't use it on either tvOS or osx.

Reply:
Thanks for clarifying!